cpp/src/parquet/encoding.cc

```diff
@@ -3265,6 +3420,14 @@ std::unique_ptr<Encoder> MakeEncoder(Type::type type_num, Encoding::type encodin
       default:
         throw ParquetException("DELTA_LENGTH_BYTE_ARRAY only supports BYTE_ARRAY");
     }
+  } else if (encoding == Encoding::DELTA_BYTE_ARRAY) {
+    switch (type_num) {
+      case Type::BYTE_ARRAY:
+        return std::make_unique<DeltaByteArrayEncoder<ByteArrayType>>(descr, pool);
+      default:
+        throw ParquetException("DELTA_BYTE_ARRAY only supports BYTE_ARRAY");
```

Comment:
Seems standard says it support FIXED_LENGTH_BYTE_ARRAY. We can add a todo here?

Reply:
Added!

Reply:
Great work, though it seems that we can change the error message here?

Reply:
Done.